service/tbc/rpc.go

```diff
@@ -259,18 +294,21 @@ func (s *Server) handleUtxosByAddressRequest(ctx context.Context, req *tbcapi.Ut
 			e := protocol.NewInternalError(err)
 			return &tbcapi.UtxosByAddressResponse{
 				Error: e.ProtocolError(),
-			}, e
+			}, err
 		}
 
-		// TODO(joshuasing): do not include raw err in errors sent to client
 		return &tbcapi.UtxosByAddressResponse{
-			Error: protocol.RequestErrorf("error getting utxos for address: %s", err),
-		}, nil
+			Error: protocol.RequestErrorf("error getting utxos for address: %s", req.Address),
+		}, err
```

Comment:
I think we could return `nil` as the error here, as this error is likely caused by the client.
What do you think?

Reply:
ah yes, I'll update